app/uk/gov/hmrc/soletraderidentificationfrontend/services/AuditService.scala

```diff
@@ -19,15 +19,23 @@ package uk.gov.hmrc.soletraderidentificationfrontend.services
 import play.api.libs.json.Json
 import uk.gov.hmrc.http.{HeaderCarrier, InternalServerException}
 import uk.gov.hmrc.play.audit.http.connector.AuditConnector
+import uk.gov.hmrc.soletraderidentificationfrontend.config.AppConfig
 import uk.gov.hmrc.soletraderidentificationfrontend.models.IndividualDetails
+import uk.gov.hmrc.soletraderidentificationfrontend.models.{JourneyConfig, Registered, RegistrationFailed, RegistrationNotCalled}
 
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class AuditService @Inject()(auditConnector: AuditConnector, soleTraderIdentificationService: SoleTraderIdentificationService) {
+class AuditService @Inject()(appConfig: AppConfig,
+                             auditConnector: AuditConnector,
+                             soleTraderIdentificationService: SoleTraderIdentificationService) {
+
+  def auditIndividualJourney(journeyId: String, journeyConfig: JourneyConfig)
+                            (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] = {
+
+    val callingService: String = journeyConfig.pageConfig.optServiceName.getOrElse(appConfig.defaultServiceName)
```

Comment:
This service name value is likely to be something like "Register for VAT", rather than "vat-registration" as is indicated on the confluence page. It might be worth confirming that this is the value we want to send.

Reply:
I checked with Liz and she said she wanted the optServiceName value or its default. This did cause quite a bit of reworking as this value was not available in the audit service.